firstvoices/firstvoices/celery.py

```diff
@@ -17,18 +16,9 @@
 
 @app.task(bind=True)
 def debug_task(self):
-    print(f"Request: {self.request!r}")
+    log.info(f"Request: {self.request!r}")
 
 
 @app.task(ignore_result=True)
 def link_error_handler(request, exc, traceback):
     log.error(f"Task {request.id} failed\n{exc}")
-
-
-@app.on_after_finalize.connect
-def setup_periodic_tasks(sender, **kwargs):
-    # print a debug message every 3 minutes
-    sender.add_periodic_task(
-        crontab(minute="*/3"),
-        debug_task.s(),
-    )
```

Comment:
Is this no longer necessary? Was it being used for something?

Reply:
Yes, it was used to print a debug message every 3 minutes, that I confirmed with Rohit we do not require. This can also help keep the logs clean.